Sniffs/WhiteSpace/PHPOpenTagSniff.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace ONGR\Sniffs\WhiteSpace;
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+
+use PHP_CodeSniffer_File;
```

Comment:
License header should be alike with the one in other files, just with ONGR Team author tag. Also namespace should go after the license header.
And I think it's time to report a new bug, namespace and use statements have 4 blank lines between them.